.buildkite/cd/config.yml

```diff
@@ -0,0 +1,16 @@
+name: ray-release
+artifacts_bucket: ray-ci-artifact-branch-public
+ci_temp: s3://ray-ci-artifact-branch-public/ci-temp/
+ci_work_repo: 029272617770.dkr.ecr.us-west-2.amazonaws.com/rayproject/citemp
+forge_prefix: cr.ray.io/rayproject/
+builder_queues:
+  builder: builder_queue_branch
+runner_queues:
+  default: release_queue_small
+buildkite_dirs:
+  - .buildkite/cd
```

Comment:
could you change the dir name to `release` or `release_tests` ? I feel that `cd` is quite ambiguous if not used with `ci` together.